tests/main/security-dev-input-event/task.yaml

```diff
@@ -0,0 +1,68 @@
+summary: Ensure that /dev/input/event* is denied by default.
+
+details: |
+    The default policy disallows access to /dev/input/event*.
+
+    The joystick interface disallows access to /dev/input/event* for
+    non-joysticks.
+
+    The test checks the snap is not able to access /dev/input/event* with or
+    without the joystick interface connected. We do this since the
+    /dev/input/event* devices are sensitive and because the joystick interface
+    adds a /dev/input/event* AppArmor glob rule that relies entirely on the
+    device cgroup for enforcement.
+
+prepare: |
+    echo "Given the test-snapd-event snap is installed"
+    snap install test-snapd-event
+
+restore: |
+    rm -f call.error
+    snap remove test-snapd-event
+
+execute: |
+    echo "The joystick plug is not connected by default"
+    snap interfaces -i joystick | MATCH "\- +test-snapd-event:joystick"
+
+    if [ "$(snap debug confinement)" != "strict" ]; then
+        exit 0
+    fi
+
+    # Detect a keyboard by looking in /dev/input/by-path to find the first
+    # '-event-kbd'
+    evdev_kbd=$(ls -l /dev/input/by-path/*-event-kbd | head -1 | sed 's#.* -> ../##')
+    if [ -z "$evdev_kbd" ]; then
+        # Fail hard if no keyboard so as not to mask detection problems
```

Comment:
Could you please echo this, so it is easier to understand in the logs why the test failed.

Reply:
Done